registration-processor/core-processor/registration-processor-uin-generator-stage/src/main/java/io/mosip/registration/processor/stages/uingenerator/stage/UinGeneratorStage.java

```diff
@@ -532,6 +543,49 @@ private IdResponseDTO sendIdRepoWithUin(String regId, String process, JSONObject
 		return result;
 
 	}
+	
+	private void idResponseProcessUtility(IdResponseDTO idResponseDTO, InternalRegistrationStatusDto registrationStatusDto,
```

Comment:
This method name sounds like noun, but methods are suppose to be named as verbs.
I believe the creation of this method is for restructuring the code, the thought is good, but lets not do the restructuring in the bug fixes of patch releases. Here very minimal change to the code is required since the patch changes will not go through exhaustive QA validation.

Reply:
Please try the restructuring part when code is fixed in develop

Reply:
Please create one more method with handleForceMerge with line number 276 to 290